tests/apps/actordotnet/Program.cs

```diff
@@ -0,0 +1,30 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace DaprDemoActor
+{
+  using Dapr.Actors.AspNetCore;
+  using Microsoft.AspNetCore.Hosting;
+  using Microsoft.Extensions.Hosting;
+
+  public class Program
+  {
+    public static void Main(string[] args)
+    {
+      CreateHostBuilder(args).Build().Run();
+    }
+
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+        Host.CreateDefaultBuilder(args)
+            .ConfigureWebHostDefaults(webBuilder =>
+            {
+              webBuilder.UseStartup<Startup>().UseActors(options =>
+              {
+                options.Actors.RegisterActor<CarActor>();
+              })
+              .UseUrls("http://*:3000");
```

Comment:
`UseUrls` is kinda extra here, I won't push back if you have a reason to keep it, but it's wierd to change the behavior for no reason.

Reply:
It needs to listen to all addresses, otherwise the pod does not respond. I can move this to dockerimage cmd but is needed.

Reply:
I was referring to the port, makes sense.